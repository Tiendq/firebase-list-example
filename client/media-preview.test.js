import React from 'react';
import { render } from 'react-testing-library';
import MediaPreview from './media-preview';

test('ImagePreview renders correctly', () => {
  let { container } = render(<MediaPreview mediaType="image" url="http://localhost/test.jpg" />);
  expect(container.firstChild).toMatchSnapshot();
});

test('AudioPreview renders correctly', () => {
  let { container } = render(<MediaPreview mediaType="audio" url="http://localhost/test.wav" />);
  expect(container.firstChild).toMatchSnapshot();
});

test('VideoPreview renders correctly', () => {
  let { container } = render(<MediaPreview mediaType="video" url="http://localhost/test.mpg" />);
  expect(container.firstChild).toMatchSnapshot();
});
