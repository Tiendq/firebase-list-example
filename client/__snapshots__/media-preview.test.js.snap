// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AudioPreview renders correctly 1`] = `
<audio
  class="audio-preview"
  controls=""
  src="http://localhost/test.wav"
/>
`;

exports[`ImagePreview renders correctly 1`] = `
<img
  class="image-preview"
  src="http://localhost/test.jpg"
/>
`;

exports[`VideoPreview renders correctly 1`] = `
<video
  class="video-preview"
  controls=""
  src="http://localhost/test.mpg"
/>
`;
